text/0000-ember-data-packages.md

```diff
@@ -0,0 +1,464 @@
+ some users use very little. Providing these packages helps to clearly show the cost of various features, and better
+ allows us to enable end users to eliminate unneeded packages.
+
+Users that implement their own adapter or serializers today must still carry the significant weight of the adapter and
+ serializer implementations that `ember-data` ships regardless. This is a weight we should enable these users to eliminate.
+
+With the landing of `RecordData` and the merging of the `modelFactoryFor` RFC, it is likely that many applications
+ will soon require far less of `ember-data` than they do today. `ember-m3` is an example of a project that utilizes these
+ APIs in a way that requires significantly less of the `ember-data` experience.
+
+**Provide Infrastructure for Additional Changes**
+
+`ember-data` is entering a period of extended evolution, of which `RecordData` and `modelFactoryFor` are only the early
+  pieces. For example, current thinking includes the possibility of `ember-data` evolving to provide an `ember-m3`-like
+  experience for `json-api` as the default out-of-the-box experience, and a rethinking of how we manage the request/response
+  lifecycle when fulfilling a request for data.
+  
+ These experiences would live alongside the existing experience for a time prior to any deprecations of the current layer,
+  and it is possible that sometimes the current experience would never be deprecated. Subdividing `ember-data` into these
+  packages will enable us to provide a more seamless transition between these experiences without hoisting any package
+  size costs onto users that do not use either the current or the new experience.
+
+**Improve our CI Time**
+
+Currently `ember-data` lives in the `emberjs` organization on `github` despite owning the `ember-data` organization.
+  Other core projects (`ember-cli`, `glimmer`, `ember-learn`) have already established the value of a core team utilizing
+  their own organization. This includes improvements in managing membership permissions for their respective teams.
+  
+Today, presence in the `emberjs` organization on `github` also places `ember-data` in the
+  `emberjs` organization on `Travis`, where we compete for a shared pool of test instances, often delaying our tests
+  by as much as an extra hour. A move to the `ember-data` organization would move our `CI` runs out of the `emberjs`
+  on `Travis` into our own organization, removing us from the shared pool. 
+
+## Detailed design
+
+<table>
+  <thead>
+    <tr>
+      <th colspan="2">Before</th>
+      <th>After</th>
+    </tr>
+    <tr>
+        <th>import DS from 'ember-data';</th>
+        <th>Direct Import</th>
+        <th>New Location</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td colspan="3"><h3>@ember-data/model</h3></td>
+    </tr>
+    <tr>
+      <td>DS.Model</td>
+      <td>import Model from 'ember-data/model';</td>
+      <td>import { Model } from '@ember-data/model';</td>
```

Comment:
I feel like this should stay as the default export. This would mirror the pattern from #176

Reply:
@chadhietala I could agree there, especially as the other named packages do have a default export. The reasoning for `Model` specifically to not have one was that the common case is

```ts
import { Model, attr, belongsTo, hasMany } from '@ember-data/model';
```

but this seems equally fine and less likely to be confusing since it holds the same pattern as elsewhere.

```ts
import Model, { attr, belongsTo, hasMany } from '@ember-data/model';
```